### ISSUE TITLE
File hierarchy.

### ISSUE THREAD

**Issue by cmhamiche** [ilia3101/MLV-App#191]
It's a feature request.
Can I suggest putting dfort's focus pixels maps in a separate folder (eventually with his permission, give users the option to get a build with the whole repo or with none of them) and lastly put other executables in a separated folder so users ain't directly exposed to them.  
No one can presume that users won't inadvertently click on them.
In such cases raw2mlv will produce a gigantic file depending on the folder (with all of the focus pixel masp and the current mlv build, it's a 15,8 gb borked mlv... you'll get complaints about it sooner or later).
It's not something often required but it's not often that a software produces such big files and it's probably a quicker fix to change some paths than performing data validation in raw2mlv.

**Comment by masc4ii** [comment]
Normally, nobody has to click any file in MLVApp folder. And only on Windows you could start via click. Focus pixel maps are installed via drag and drop, no need to open the folder. RAW2MLV will be included one day (if Ilia gives his okay), after the current bugs are solved. For MLVApp.exe one can create a link on Windows, on other platforms there is even no need for this. Maybe we could build an installer for Windows - also in past I often saw, that people fail to copy a folder from a to b. 
Sometimes fpm's change very often, sometimes you need an older one. It is no good idea to include some files - nobody knows if you have the right ones.

**Comment by cmhamiche** [comment]
You're right about the focus pixels maps inclusion and sure having an installer would be a good thing. I got you.
